client/state/audio/middleware.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_RECEIVE_EVENT,
+} from 'state/action-types';
+
+const isAudioSupported = () => typeof window === 'object' && typeof window.Audio === 'function';
```

Comment:
Would this ever change in a single page view? If you need to stub it in tests you can do something like the following or stub the window.Audio property.

```js
export const isAudioSupported  = typeof window === 'object' && typeof window.Audio === 'function';
```